client/components/TestRenderer/TestRenderer.module.css

```
@@ -167,10 +167,26 @@ label {
 
   &.untestable-label {
     margin-bottom: var(--8px);
+    position: relative;
 
     input {
       margin-right: var(--4px);
     }
+
+    &:hover [role='tooltip'],
+    input:focus + [role='tooltip'] {
+      visibility: visible;
+    }
+
+    [role='tooltip'] {
+      position: absolute;
+      top: 100%;
+
+      background-color: rgba(255, 255, 255, 0.9);
+      padding: 1em;
+      border-color: #999;
```

```suggestion
      border-color: var(--border-gray);
```
^ for consistency or if you'd really prefer that color, you could define it in the `index.css`.

Missing a border style here as well to show this